xml/System/Half.xml

```diff
@@ -165,8 +165,9 @@
       </Parameters>
       <Docs>
         <param name="obj">To be added.</param>
-        <summary>Returns a value that indicates whether this instance is equal to a specified <paramref name="obj" />.</summary>
-        <returns>To be added.</returns>
+        <summary>Returns a value that indicates whether this instance is equal to the specified <paramref name="obj" />.</summary>
+        <returns>
+          <see langword="true" /> if <paramref name="obj" /> is an instance of <seealso cref="T:System.Half" /> and equals the value of this instance; otherwise, <see langword="false" />.</returns>
```

Comment:
```suggestion
          <see langword="true" /> if <paramref name="obj" /> is an instance of <see cref="T:System.Half" /> and equals the value of this instance; otherwise, <see langword="false" />.</returns>
```